src/main/java/pl/uj/edu/JImageStream/api/FileCollector.java

```diff
@@ -0,0 +1,29 @@
+package pl.uj.edu.JImageStream.api;
```

Comment:
We should move this to other package, something like pl.uj.edu.JImageStream.api.collectors would be fine